app/Http/Controllers/PurchaseOrderController.php

```diff
@@ -8,8 +8,7 @@ class PurchaseOrderController extends Controller
 {
     public function getPurchaseOrder()
     {
-        $purchaseOrders = PurchaseOrder::orderBy('created_at', 'desc')->paginate(10);
-    
+        $purchaseOrders = PurchaseOrder::with('supplier')->orderBy('created_at', 'desc')->paginate(10);
```

Comment:
sebisa mungkin kode eluqoent/query buildir dibuat di model.
Controller hanya memanggil ke model saja.

Reply:
Baik pak ini perubahan pada **Controller** 

```suggestion
        $purchaseOrders = PurchaseOrder::getAllPurchaseOrders();
        return view('purchase_orders.list', compact('purchaseOrders'));
```

Perubahan pada **Model**

```
public function supplier() {
        return $this->belongsTo(Supplier::class, 'supplier_id', 'supplier_id');
    }

    public static function getAllPurchaseOrders()
    {
        return self::with('supplier')->orderBy('created_at', 'desc')->paginate(10);
}
```